### PR TITLE
LGA-1940-Update Ingress

### PR DESCRIPTION
**Pull request by jogilder**
Updated version in ingress, still has references to live1 so unlikely to pass tests. Also are references to live1 under kubernetes_deploy folders

## What does this pull request do?

Required.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Comment by said-moj** [comment]
Sorry I think I forgot to remove the staging-live1 and production-live1 folders. These can now be removed

**Review by jogilder — COMMENTED** [review]
OK, My plan would also be to remove lines 11 and 12 and 17 and 18 in here and the equivalent in staging.yml and the same files under kubernetes_deploy. I will do that and recommit.

File: kubernetes_deploy/production/deployment.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ spec:
           name: http
         env:
         - name: ALLOWED_HOSTS
-          value: ".laa-fala-production.apps.live-1.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk"
+          value: ".laa-fala-production.apps.live.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk"
```
Comment (said-moj) [inline kubernetes_deploy/production/deployment.yml:40]: I think this needs to stay as is as those live-1 domains still work on the new cluster

Comment (jogilder) [reply kubernetes_deploy/production/deployment.yml:40]: ok, does it also need to reference live or just live-1? And what about in the ingress files?

**Pull request merged**